src/EnergyPlus/AirflowNetworkBalanceManager.cc

```diff
@@ -5157,6 +5255,283 @@ namespace AirflowNetworkBalanceManager {
 		return CalcWindPressure;
 	}
 
+	Real64
+	AirDensity(
+		Real64 T // Temperature in Celsius
+	)
+	{
+		// Dry air density {kg/m3}
+		// Correlated over the range -20C to 70C
+		// Reference Cengel & Ghajar, Heat and Mass Transfer. 5th ed.
+
+		Real64 const LowerLimit = -20;
+		Real64 const UpperLimit = 70;
+
+		Real64 const a = 1.292;
+		Real64 const b = -0.00467558800659982;
+		Real64 const c = 0.0000125483407801669;
+
+		if ( T < LowerLimit ) {
+			ShowWarningMessage( "Air temperature out of limits for density calculation");
+			return a + b * LowerLimit + c * pow_2( LowerLimit );
+		} else if ( T > UpperLimit ) {
+			ShowWarningMessage( "Air temperature out of limits for density calculation");
+			return a + b * UpperLimit + c * pow_2( UpperLimit );
+		} else {
+			return a + b * T + c * pow_2( T );
+		}
+	}
+
+	Real64
+	AirThermConductivity(
+		Real64 T // Temperature in Celsius
+	)
+	{
+		// Dry air thermal conductivity {W/m-K}
+		// Correlated over the range -20C to 70C
+		// Reference Cengel & Ghajar, Heat and Mass Transfer. 5th ed.
+
+		Real64 const LowerLimit = -20;
+		Real64 const UpperLimit = 70;
+
+		Real64 const a = 0.02364;
+		Real64 const b = 0.0000754772569209165;
+		Real64 const c = -2.40977632412045e-8;
+
+		if ( T < LowerLimit ) {
+			ShowWarningMessage( "Air temperature out of limits for conductivity calculation");
+			return a + b * LowerLimit + c * pow_2( LowerLimit );
+		} else if ( T > UpperLimit ) {
+			ShowWarningMessage( "Air temperature out of limits for conductivity calculation");
+			return a + b * UpperLimit + c * pow_2( UpperLimit );
+		} else {
+			return a + b * T + c * pow_2( T );
+		}
+	}
+
+	Real64
+	AirCp(
+		Real64 T // Temperature in Celsius
+	)
+	{
+		// Dry air specific heat {J/kg-K}
+		// Correlated over the range -20C to 70C
+		// Reference Cengel & Ghajar, Heat and Mass Transfer. 5th ed.
+
+		Real64 const LowerLimit = -20;
+		Real64 const UpperLimit = 70;
+
+		Real64 const a = 1006;
+		Real64 const b = 0.0431668570326241;
+		Real64 const c = -0.00042378743454105;
+
+		if ( T < LowerLimit ) {
+			ShowWarningMessage( "Air temperature out of limits for specific heat calculation");
+			return a + b * LowerLimit + c * pow_2( LowerLimit );
+		} else if ( T > UpperLimit ) {
+			ShowWarningMessage( "Air temperature out of limits for specific heat calculation");
+			return a + b * UpperLimit + c * pow_2( UpperLimit );
+		} else {
+			return a + b * T + c * pow_2( T );
+		}
+	}
+
+	Real64
+	AirDynamicVisc(
```

Comment:
This function and `AirKinematicVisc` will class with the rest of the code which is using a different correlation for viscosity. See [AirflowNetworkSolver.cc, line 538](https://github.com/NREL/EnergyPlus/blob/develop/src/EnergyPlus/AirflowNetworkSolver.cc#L538). It is pretty similar, but I think we should stick with that one.